From 727e6a9185033198c57774b894fae3419a035169 Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Mon, 17 Dec 2018 08:51:37 +0800
Subject: [PATCH 02/25] arm: dts: zynq: active watchdog status explicitly

The latest zc706 has transferred to this change, so we do the similar
updates as well.

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
(cherry picked from commit 4d34c7ab5d108e2da098968197bf5420ca48f426)
---
 arch/arm/boot/dts/zynq-zc706.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/zynq-zc706.dts b/arch/arm/boot/dts/zynq-zc706.dts
index cc18cb7af3c7..aa2c58872bd5 100644
--- a/arch/arm/boot/dts/zynq-zc706.dts
+++ b/arch/arm/boot/dts/zynq-zc706.dts
@@ -366,5 +366,6 @@
 };
 
 &watchdog0 {
+	status = "okay";
 	reset-on-timeout;
 };
-- 
2.17.0

